readme.md

```diff
@@ -16,6 +18,7 @@ See [Guido wiki](https://github.com/grame-cncm/guidolib/wiki) for building instr
 ---
 
 Travis build status:  <a href="https://travis-ci.org/grame-cncm/guidolib"><img src="https://travis-ci.org/grame-cncm/guidolib.svg?branch=dev"></a>
+GitHub Actions docker build: ![CI](https://github.com/olafure/guidolib/workflows/CI/badge.svg?branch=ci-with-gh-actions)
```

Comment:
This will need to be changed to the correct repo & branch

Reply:
yes, just replace olafure with grame-cncm